crates/core/src/factories/core_publish/test.rs

```diff
@@ -13,9 +14,60 @@ use crate::{
 
 use super::CorePublishConfig;
 
+const SIG: &[u8] = b"fake-signature";
+const INVALID_SIG: &[u8] = b"invalid-fake-signature";
+
+#[derive(Debug)]
+struct TestCrypto;
```

Comment:
There's a [TestLocalAgent](https://github.com/holochain/kitsune2/blob/c880ae72e343935697efc3b5b500e67a06d5092f/crates/test_utils/src/agent.rs#L35) in test_utils like this... But I'm also fine with this being inline to contrast with the TestCryptoInvalid below.

Reply:
Okay, addressed this as well. Please have a look.